msticnb/nb/azsent/host/host_logons_summary.py

```diff
@@ -153,9 +153,14 @@ def run(
                     f"Could not find event records for host {value}. "
                     + "Results may be unreliable."
                 )
-                return self._last_result
-            host_type = host_name[1] or None
-            host_name = host_name[0] or value
+            try:
```

Comment:
I'm going to change the way that validate host name returns values - it's getting a bit messy. Will do this in my account summary branch